packages/strapi-admin/admin/src/icons/Duplicate.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Duplicate = ({ fill, ...rest }) => (
+  <svg width="13" height="13" xmlns="http://www.w3.org/2000/svg" {...rest}>
+    <path
+      d="M12.304 2.786a.67.67 0 01.493.203c.135.135.203.3.203.493v8.822a.672.672 0 01-.203.493.672.672 0 01-.493.203H5.339a.672.672 0 01-.493-.203.672.672 0 01-.203-.493v-2.09H.696a.672.672 0 01-.493-.203A.672.672 0 010 9.518V4.643c0-.194.048-.406.145-.639.097-.232.213-.415.348-.55l2.96-2.96a1.7 1.7 0 01.551-.349C4.237.048 4.45 0 4.643 0H7.66c.193 0 .358.068.493.203.135.136.203.3.203.493v2.38c.329-.194.639-.29.929-.29h3.018zM8.357 4.33L6.188 6.5h2.17V4.33zM3.714 1.545l-2.169 2.17h2.17v-2.17zM5.136 6.24L7.43 3.946V.93H4.643v3.017a.672.672 0 01-.203.494.672.672 0 01-.494.203H.93v4.643h3.714V7.429c0-.194.048-.407.145-.639.097-.232.213-.416.348-.551zm6.935 5.832V3.714H9.286v3.018a.672.672 0 01-.203.493.672.672 0 01-.494.204H5.571v4.642h6.5z"
+      fill={fill}
+      fillOpacity=".75"
+    />
+  </svg>
+);
+
+Duplicate.defaultProps = {
+  fill: '#0E1622',
+};
+
+Duplicate.propTypes = {
+  fill: PropTypes.string,
+};
+
+export default Duplicate;
```

Comment:
Is it the same icon?
https://github.com/strapi/buffet/blob/develop/packages/buffetjs-icons/src/components/Duplicate/index.js

Reply:
Yes it is I'll update it with the one front buffet!